From 228044ce82779b7210b0fe757f86794127822fa4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Milan=20P=2E=20Stani=C4=87?= <mps@arvanta.net>
Date: Thu, 3 Aug 2023 14:02:00 +0200
Subject: [PATCH] revert bump libudev version

need it for Alpine Linux to not be selected by builder instead of eudev
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 4e970b8..58f70e4 100644
--- a/Makefile
+++ b/Makefile
@@ -43,7 +43,7 @@ libudev.pc: libudev.pc.in
 		-e 's|@exec_prefix@|${PREFIX}|g' \
 		-e "s|@libdir@|$$libdir|g" \
 		-e "s|@includedir@|$$includedir|g" \
-		-e 's|@VERSION@|251|g' \
+		-e 's|@VERSION@|243|g' \
 		libudev.pc.in > libudev.pc
 
 install-headers: udev.h
-- 
2.41.0

